.github/merge_rules.json

```diff
@@ -41,6 +41,22 @@
       "approved_by": ["linbinyu", "kit1980", "IvanKobzarev", "dreiss"],
       "mandatory_checks_name": ["Facebook CLA Check", "Lint"]
    },
+   {
+      "name": "FFT",
+      "patterns": [
+         "aten/src/ATen/native/cuda/*FFT*.h",
```

Comment:
There are also these "SpectralOps" files which contain the operator implementations:
```cpp
         "aten/src/ATen/native/SpectralOps.cpp",
         "aten/src/ATen/native/mkl/SpectralOps.cpp",
         "aten/src/ATen/native/cuda/SpectralOps.*",
```